refinery/ui/source/js/dashboard/controllers/tutorial-data-upload-second-page.js

```diff
@@ -20,12 +20,10 @@ function dataUploadTutorialCtrl2 (
   $scope.dataUploadExitEvent = function () {
   };
 
-  $scope.dataUploadChangeEvent = function (event) {
-    lastEventId = event.id;
```

Comment:
yeah: The variable is never used, so seemed good to get rid of it... but there's another no-op just above, so it doesn't seem terrible to add another: Always possible that something looks for some function by that name.